### PR TITLE
Allow embeds from JSFiddle

### PR DESCRIPTION
**Pull request by srukle**
adds JSFiddle to whitelist for issue #1713

### CODE REVIEW

File: cgi-bin/DW/Hooks/EmbedWhitelist.pm [hunk 1]
```diff
@@ -65,6 +65,8 @@ my %host_path_match = (
     "docs.google.com"       => [ qr!^/(document|spreadsheets?|presentation)/!, 1 ],
     
     "books.google.com"      => [ qr!^/ngrams/!, 1 ],
+    
+    "jsfiddle.net"          => [ qr!^/.*/embedded/$!, 1 ],
```
Comment (kareila) [inline cgi-bin/DW/Hooks/EmbedWhitelist.pm]: Using `.*` in regexps is usually a bad idea, causing the computer to do a lot of extra work to match characters we don't care about.  Just match the end of the path: `qr!/embedded/$!`

**Comment by kareila** [comment]
It's also a good idea to add a test case for the new site to t/embed-whitelist.t to make sure things are working as expected.

Thanks :)

**Comment by srukle** [comment]
Updated whitelist entry and tested embedded example in dreamhack.

Example in initial request works with changes to whitelist entry as suggested in September within dreamhack environment.
https://www.dreamwidth.org/support/see_request?id=32942

**Comment by spinda** [comment]
Any hope of this getting merged?

**Comment by kareila** [comment]
I should have time to take a look at it soon!

**Comment by kareila** [comment]
So, there is one thing still preventing this from getting merged: the second element of the array, the '1' has been removed and should be readded.  This signals whether or not the given site supports https and this one does.

**Comment by kaberett** [comment]
@kareila ooh, that's not mentioned explicitly at http://wiki.dreamwidth.net/wiki/index.php/Adding_Embed_Whitelist_Items -- might be good to add it? Will try to remember to get to it...

**Comment by srukle** [comment]
@kareila, change was made today.

**Comment by kareila** [comment]
@kaberett wiki is updated - I didn't know we had a page for these :)

@srukle thanks for updating this so quickly!

**Comment by kaberett** [comment]
@kareila thank you!

@srukle thank you also :-)

**Pull request merged**